src/services/doppler-legacy-client.ts

```diff
@@ -454,13 +499,13 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
     return mapHeaderDataJson(response.data);
   }
 
-  public async getAllPlans(): Promise<PlanModel[]> {
+  public async getAllPlans(): Promise<any> {
```

Comment:
I think that declaring that you are returning _any_ is not a good idea because your plans are already sanitized and mapped. You already know what is the type of the result, why losing that information with _any_?

Reply:
This is the method when information is yet to be sanitized. Before it was not.